rocky/tools/ooi_form.py

```diff
@@ -61,7 +61,7 @@ def generate_form_fields(
             elif issubclass(field.type_, Enum):
                 fields[name] = generate_select_ooi_type(field)
             elif issubclass(field.type_, str):
-                fields[name] = forms.CharField(max_length=256, **default_attrs)
+                fields[name] = forms.CharField(max_length=2056, **default_attrs)
```

Comment:
Practical user agent implementations have limits on the number and
   size of cookies that they can store.  General-use user agents SHOULD
   provide each of the following minimum capabilities:
   
   At least 4096 bytes per cookie (as measured by the sum of the
      length of the cookie's name, value, and attributes).

Reply:
Okay, should I change it to 4096? This is for all str fields btw

Reply:
I think so, yes. A better handling of max-lengths needs to be done by checking the input against the model's fields and their max length.